doc/sphinx/language/core/variants.rst

```diff
@@ -6,11 +6,30 @@ Variants and the `match` construct
 Variants
 --------
 
+The :cmd:`Variant` command allows defining types by listing
+the :term:`inhabitants <inhabitant>` of the type. This includes enumerated types from
+programming languages, such as Booleans, characters or even the
+degenerate cases of the unit and empty types. Variant types more
+generally include enumerated types with arguments or even enumerated
+types with parametric arguments such as option types, sum types,
+etc. It also includes predicates or type families defined by cases
+such as the Boolean reflection or equality predicates. Observing the
+form of the :term:`inhabitants <inhabitant>` of a variant type is done by case analysis
+using the :cmd:`match` expression. Each case is described by what is
+called a :gdef:`constructor` of the type. For instance, Booleans have two
+constructors: :g:`true` and :g:`false`.
```

Comment:
```suggestion
The :cmd:`Variant` command allows defining types by listing
the :term:`inhabitants <inhabitant>` of the type.  This includes enumerated types from
programming languages, such as Booleans, characters or even the
degenerate cases of the unit and empty types. Variant types more
generally include enumerated types with arguments or even enumerated
types with parametric arguments such as option types and sum types.
It also includes predicates or type families defined by cases
such as the Boolean reflection or equality predicates. Observing the
form of the :term:`inhabitants <inhabitant>` of a variant type is done by case analysis
using the :cmd:`match` expression. Each case is described by what is
called a :term:`constructor` of the type. For instance, Booleans have two
constructors: :g:`true` and :g:`false`.
```

Reply:
Adopting "etc" part.

For constructor, don't we need `gdef` to define it?

Reply:
Yes, `gdef`.  [Homer nods :-)](https://en.wiktionary.org/wiki/even_Homer_nods#English)